spec/spec_helper.rb

```diff
@@ -38,6 +36,11 @@
   config.include BulkDataHelper
   config.include Capybara::RSpecMatchers, type: :request
 
+  unless config.files_to_run.one?
```

Comment:
This is an improvement 👍 I suppose we could look at only running it on full test suite by doing something like:

```suggestion
  if ARGV.grep(/rake$/)
```

But this is more brittle 🤷‍♂ I'm happy going with this compromise for the time being!

Reply:
Yeah, I also thought it was brittle. Especially as I usually want to look at coverage by running `rspec` 🤔.

Reply:
This looks cool 👍